### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Abdona**
New York Times Articles.html and CSS were used in this project. I used Flexes and grids to organize the webpage.

### CODE REVIEW

File: index.html
Comment (teekaytech) [inline index.html:32]: It is better to use font awesome icons here instead of the SVG icons. You've already added font awesome CDN link to your project. I suggest you use font awesome icons.

If you have no other option than to use SVG icons, kindly move them to separate files. Dumping them directly in the HTML file is not ideal: it messes up your code easily and also makes it difficult to manage your code either by you or others.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Abdona :wave:

__You've made laudable progress in this milestone.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Kindly update your Readme file. You can use [this template](https://github.com/microverseinc/readme-template) provided by microverse as a guide.
- [x] You can use a more descriptive title for your Pull Request. Titles like `New York Times`, `New York Times Article`, `NYT Feature`, or similar can be used.
- [x] You can improve on the directory structures of your project. For instance, You can have an `assets` directory/folder where you put assets/resources like images, CSS files, etc. Then inside the `assets` folder, you can create different folders for different assets. e.g. `images` folder to store the images, `css` folder to store stylesheets, etc.
- [x] Some elements are missing in the navbar. Check the images below, the navigation bar should have:
- a "Hamburger button" on the leftmost side 
- a 'Log in' button on the rightmost side 

**Original Navbar**
original

**Your Clone**
clone

- [x] The `Editors’ Picks` section on the right side is missing. Please implement it.

**Original**
original

**Your Clone**
clone

- [x] You might want to recheck the `Theory of Inflation` section of your webpage. It seems to be having some issues. Check the images below:

**Original**
original

**Your Clone**
clone


- [x] Be careful of your usage of id selectors for CSS rules. You should strive to get used to using class selectors extensively for CSS rules.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by Abdona** [comment]
Hello @teekaytech thank you for your reviews.Regarding the Inflation image, in the middle of the article.I send you an attachment, it shows that it works well on my side.I do not know why it's messed up on your side.
Abdulrahman Naser


    On Friday, February 19, 2021, 08:10:08 PM GMT+2, Taofeek Olalere <notifications@github.com> wrote:  
 
 


@teekaytech requested changes on this pull request.

Status: Changes Required ⚠️ 🚦
 
Hello @Abdona 👋
 
You've made laudable progress in this milestone.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.

[ REQUIRED CHANGES ]
   
   -  Kindly update your Readme file. You can use this template provided by microverse as a guide.
   -  You can use a more descriptive title for your Pull Request. Titles like New York Times, New York Times Article, NYT Feature, or similar can be used.
   -  You can improve on the directory structures of your project. For instance, You can have an assets directory/folder where you put assets/resources like images, CSS files, etc. Then inside the assets folder, you can create different folders for different assets. e.g. images folder to store the images, css folder to store stylesheets, etc.
   -  Some elements are missing in the navbar. Check the images below, the navigation bar should have:
   - a "Hamburger button" on the leftmost side
   - a 'Log in' button on the rightmost side

Original Navbar


Your Clone

   
   -  The Editors’ Picks section on the right side is missing. Please implement it.

Original


Your Clone

   
   -  You might want to recheck the Theory of Inflation section of your webpage. It seems to be having some issues. Check the images below:

Original


Your Clone

   
   -  Be careful of your usage of id selectors for CSS rules. You should strive to get used to using class selectors extensively for CSS rules.

Please make the required changes and submit another review request when you are done.

Taofeek Olalere
Twitter | Github | @taofeekolalere on Slack
Happy Coding 😃

In index.html:
> +        <svg class="icon" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24"><path d="M9 8h-3v4h3v12h5v-12h3.642l.358-4h-4v-1.667c0-.955.192-1.333 1.115-1.333h2.885v-5h-3.808c-3.596 0-5.192 1.583-5.192 4.615v3.385z"/></svg>
+        <svg class="icon" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24"><path d="M.057 24l1.687-6.163c-1.041-1.804-1.588-3.849-1.587-5.946.003-6.556 5.338-11.891 11.893-11.891 3.181.001 6.167 1.24 8.413 3.488 2.245 2.248 3.481 5.236 3.48 8.414-.003 6.557-5.338 11.892-11.893 11.892-1.99-.001-3.951-.5-5.688-1.448l-6.305 1.654zm6.597-3.807c1.676.995 3.276 1.591 5.392 1.592 5.448 0 9.886-4.434 9.889-9.885.002-5.462-4.415-9.89-9.881-9.892-5.452 0-9.887 4.434-9.889 9.884-.001 2.225.651 3.891 1.746 5.634l-.999 3.648 3.742-.981zm11.387-5.464c-.074-.124-.272-.198-.57-.347-.297-.149-1.758-.868-2.031-.967-.272-.099-.47-.149-.669.149-.198.297-.768.967-.941 1.165-.173.198-.347.223-.644.074-.297-.149-1.255-.462-2.39-1.475-.883-.788-1.48-1.761-1.653-2.059-.173-.297-.018-.458.13-.606.134-.133.297-.347.446-.521.151-.172.2-.296.3-.495.099-.198.05-.372-.025-.521-.075-.148-.669-1.611-.916-2.206-.242-.579-.487-.501-.669-.51l-.57-.01c-.198 0-.52.074-.792.372s-1.04 1.016-1.04 2.479 1.065 2.876 1.213 3.074c.149.198 2.095 3.2 5.076 4.487.709.306 1.263.489 1.694.626.712.226 1.36.194 1.872.118.571-.085 1.758-.719 2.006-1.413.248-.695.248-1.29.173-1.414z"/></svg>
+        <svg class="icon" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24"><path d="M24 4.557c-.883.392-1.832.656-2.828.775 1.017-.609 1.798-1.574 2.165-2.724-.951.564-2.005.974-3.127 1.195-.897-.957-2.178-1.555-3.594-1.555-3.179 0-5.515 2.966-4.797 6.045-4.091-.205-7.719-2.165-10.148-5.144-1.29 2.213-.669 5.108 1.523 6.574-.806-.026-1.566-.247-2.229-.616-.054 2.281 1.581 4.415 3.949 4.89-.693.188-1.452.232-2.224.084.626 1.956 2.444 3.379 4.6 3.419-2.07 1.623-4.678 2.348-7.29 2.04 2.179 1.397 4.768 2.212 7.548 2.212 9.142 0 14.307-7.721 13.995-14.646.962-.695 1.797-1.562 2.457-2.549z"/></svg>
+        <svg class="icon" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24"><path d="M12 12.713l-11.985-9.713h23.971l-11.986 9.713zm-5.425-1.822l-6.575-5.329v12.501l6.575-7.172zm10.85 0l6.575 7.172v-12.501l-6.575 5.329zm-1.557 1.261l-3.868 3.135-3.868-3.135-8.11 8.848h23.956l-8.11-8.848z"/></svg>
+        <svg class="icon" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 24 24"><path d="M14 18l10-7.088-10-6.912v3.042s-11.618 2.583-14 12.958c5.072-5.431 14-5.218 14-5.218v3.218z"/></svg>
+        <svg class="bubbbook" stroke="none" width="16px" height="25px" viewBox="0 0 16 25" ><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g stroke="#666" fill="none"><path fill="none" d="M0.5,0.5 L0.5,24.5 L8.1716264,19.5137775 L15.5,24.4404006 L15.5,0.5 L0.5,0.5 Z"></path></g></g></svg>
+        <svg class="bubbbook" width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M12 1c-6.338 0-12 4.226-12 10.007 0 2.05.739 4.063 2.047 5.625l-1.993 6.368 6.946-3c1.705.439 3.334.641 4.864.641 7.174 0 12.136-4.439 12.136-9.634 0-5.812-5.701-10.007-12-10.007m0 1c6.065 0 11 4.041 11 9.007 0 4.922-4.787 8.634-11.136 8.634-1.881 0-3.401-.299-4.946-.695l-5.258 2.271 1.505-4.808c-1.308-1.564-2.165-3.128-2.165-5.402 0-4.966 4.935-9.007 11-9.007"/><text x="0" y="15" fill="black">615</text></svg></div></section>

It is better to use font awesome icons here instead of the SVG icons. You've already added font awesome CDN link to your project. I suggest you use font awesome icons.

If you have no other option than to use SVG icons, kindly move them to separate files. Dumping them directly in the HTML file is not ideal: it messes up your code easily and also makes it difficult to manage your code either by you or others.

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work. 💻 👀 ✨ 

- Your page design looks fine.

## However, there are some changes required 👀 🧰 🔨 🔧 
- Please fix the Live Demo link on your README file. It currently directs to a 404 missing page.
- Please edit your Pull Request description to contain a proper recount of all the new things you learned while building this project.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by krys2fa — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this project approved. 👍 

You can now merge this branch into the main branch and go on with your next project.

💡  Do not forget to submit your project completion form after merging this branch.

All the best @Abdona,
Chris

**Pull request merged**
